css/properties/ime-mode.json

```diff
@@ -12,7 +12,7 @@
               "version_added": null
             },
             "edge": {
-              "version_added": null
+              "version_added": "12"
```

Comment:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18922787/ says "Edge removed ime-mode for interoperability reasons". So maybe it was supported for a few versions, but we also need a "version_removed" here?

Reply:
The test did return true in Edge 12-18 (checked manually now) and the script I used shouldn't have added this if it had been removed again.

https://software.hixie.ch/utilities/js/live-dom-viewer/saved/6806 tests using `CSS.supports` instead, and I just confirmed it returning true in Edge 18.

It seems like perhaps they left the property but disabled the behavior.

@gked, if you are the "Grisha L." in https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18922787/, can you shed some light on what happened here?